src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -53,9 +53,15 @@ import {
 import { ExperimentData } from "../../../../../../../telemetry/generated/nimbus/experiments";
 import { PetitionBanner } from "../../../../../../components/client/PetitionBanner";
 import { useLocalDismissal } from "../../../../../../hooks/useLocalDismissal";
+import { ScanResult } from "../../../../../../functions/server/onerep";
 
 export type TabType = "action-needed" | "fixed";
 
+export type RemovalTimeEstimate = {
+  d: ScanResult["data_broker"];
+  t: number;
```

Comment:
Nit, but maybe a JSDoc comment would be useful here describing the unit of `t` - is it days?

Reply:
Actually, maybe that would be better for `DataBrokerRemovalTime` in `getDataBrokerRemovalTimeEstimates.ts`, which maybe should be re-used here?

Reply:
Oh, yes. I meant to not introduce a duplicate type: [5a9b1b8](https://github.com/mozilla/blurts-server/pull/5189/commits/5a9b1b80cc4022a900bfd612b8c01b3d574fe0e7).